website/content/api-docs/session.mdx

```diff
@@ -57,8 +57,14 @@ The table below shows this endpoint's support for
   if you override this list, you include the default `serfHealth`. Added in
   Consul 1.7.0.
 
-- `ServiceChecks` `(array<string>: nil)` - Specifies a list of service health
-  check IDs (commonly `CheckID` in API responses). Added in Consul 1.7.0.
+- `ServiceChecks` `(array<ServiceCheck>: nil)` - Specifies a list of service
+  checks. Added in Consul 1.7.0.
```

Comment:
```suggestion
  checks. Added in Consul 1.7.0. A service check as the following fields:
```

This might help clarify that these two things are the fields for a `ServiceCheck`, what do you think?

Reply:
Great suggestion! I'll add it.